pkg/helmexec/runner.go

```diff
@@ -16,7 +17,7 @@ import (
 
 // Runner interface for shell commands
 type Runner interface {
-	Execute(cmd string, args []string, env map[string]string) ([]byte, error)
+	Execute(cmd string, args []string, env map[string]string, enableLiveOutput bool) ([]byte, error)
```

Comment:
Should we add enableLiveOutput as a property of Runner instead of passing it to Execute()?

Reply:
@itscaro cc @yxxhero I believe this is the outcome of my feedback https://github.com/helmfile/helmfile/pull/286#discussion_r945221101 so I'm absolutely fine with this!